Dockerfile.build

```diff
@@ -91,6 +93,28 @@ RUN git clone https://github.com/pudelkoM/behavioral-model.git /tmp/bmv2 && \
     cp /tmp/bmv2/tools/veth* ${BMV2_INSTALL}/bin/ && \
     rm -rf /tmp/bmv2
 
+RUN apt-get install -y libssl-dev
+RUN cd /tmp && wget http://www.cmake.org/files/v3.16/cmake-3.16.0.tar.gz && \
+    tar xf cmake-3.16.0.tar.gz && \
+    cd /tmp/cmake-3.16.0 && ./bootstrap && \
+    make -j${JOBS} && make install && ldconfig && \
+    rm -f /tmp/cmake-3.16.0.tar.gz && rm -rf /tmp/cmake-3.16.0
+
+RUN apt-get install -y libjansson-dev
+ARG NIKSS_COMMIT
+ENV NIKSS_INSTALL /usr/local
+RUN git clone --recursive https://github.com/NIKSS-vSwitch/nikss.git /tmp/nikss && \
+    cd /tmp/nikss && git checkout ${NIKSS_COMMIT} && \
+    ./build_libbpf.sh && \
+    mkdir build && cd /tmp/nikss/build && \
+    cmake -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=${NIKSS_INSTALL} -DBUILD_SHARED=on .. && \
+    make -j${JOBS} && make install && ldconfig && \
+    mkdir -p ${NIKSS_INSTALL}/include/nikss && \
+    # TODO: headers should be installed by cmake above
```

Comment:
@tatry note this -> we should install headers as part of `make install` or separate `make install_headers` target

Reply:
`make install_headers` makes sense